stdlib/public/TensorFlow/Execution.swift

```diff
@@ -37,10 +37,31 @@ public enum DeviceKind {
 @inline(never)
 public func withDevice<R>(_ kind: DeviceKind, _ index: UInt = 0,
                           perform body: () throws -> R) rethrows -> R {
-  _ThreadLocalState.value.pushDevice((kind, index))
-  let result = try body()
-  _ThreadLocalState.value.popDevice()
-  return result
+  return try _ExecutionContext.global.withDevice(kind, index, perform: body)
+}
+
+/// Executes a closure, making TensorFlow operations run on a device with
+/// a specific name.
+///
+/// - Parameters:
+///   - name: Device name.
+///   - body: A closure whose TensorFlow operations are to be executed on the
+///     specified kind of device.
+///
+/// A few examples of device names from https://www.tensorflow.org/alpha/guide/using_gpu
```

Comment:
Comments starting with three slashes are considered by compiler and doc generators as documentation comments, and will be included in user API docs. However, the inclusion of first-person pronouns makes it less public-facing. Would you like to switch to two slashes from this line to line 60?

Reply:
Reworded away from first-person pronouns.